src/explore-education-statistics-admin/src/pages/users/PreReleaseUsersPage.tsx

```diff
@@ -0,0 +1,79 @@
+import ButtonText from '@common/components/ButtonText';
+import Link from '@admin/components/Link';
+import Page from '@admin/components/Page';
+import userService from '@admin/services/users/service';
+import { UserStatus } from '@admin/services/users/types';
+import React, { useEffect, useState } from 'react';
+
+interface Model {
+  users: UserStatus[];
+}
+
+const PreReleaseUsersPage = () => {
+  const [model, setModel] = useState<Model>();
+
+  useEffect(() => {
+    userService.getPreReleaseUsers().then(users => {
```

Comment:
Same as earlier. Would suggest using the `useAsyncRetry` hook

Reply:
Have we got any simple examples of this? I had a look but tbh they all looked over my head for my knowledge of react at the moment. Happy for somebody else to have a look but given this page will have limited use dont want this hung up to long.

Reply:
useAsyncRetry implemented